pages/reference.mdx

```diff
@@ -626,19 +626,20 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 ###### Formats
 
-| Type                    | Structure                      | Example                         |
-| ----------------------- | ------------------------------ | ------------------------------- |
-| User                    | `<@USER_ID>`                   | `<@80351110224678912>`          |
-| User ^1^                | `<@!USER_ID>`                  | `<@!80351110224678912>`         |
-| Channel                 | `<#CHANNEL_ID>`                | `<#103735883630395392>`         |
-| Role                    | `<@&ROLE_ID>`                  | `<@&165511591545143296>`        |
-| Slash Command ^2^       | `</NAME:COMMAND_ID>`           | `</airhorn:816437322781949972>` |
-| Standard Emoji          | Unicode Characters or `:NAME:` | 💯 or `:100:`                   |
-| Custom Emoji            | `<:NAME:ID>`                   | `<:mmLol:216154654256398347>`   |
-| Custom Emoji (Animated) | `<a:NAME:ID>`                  | `<a:b1nzy:392938283556143104>`  |
-| Unix Timestamp          | `<t:TIMESTAMP>`                | `<t:1618953630>`                |
-| Unix Timestamp (Styled) | `<t:TIMESTAMP:STYLE>`          | `<t:1618953630:d>`              |
-| Guild Navigation        | `<id:TYPE>`                    | `<id:customize>`                |
+| Type                    | Structure                      | Example                                           |
+| ----------------------- | ------------------------------ | ------------------------------------------------- |
+| User                    | `<@USER_ID>`                   | `<@80351110224678912>`                            |
+| User ^1^                | `<@!USER_ID>`                  | `<@!80351110224678912>`                           |
+| Channel                 | `<#CHANNEL_ID>`                | `<#103735883630395392>`                           |
+| Role                    | `<@&ROLE_ID>`                  | `<@&165511591545143296>`                          |
+| Slash Command ^2^       | `</NAME:COMMAND_ID>`           | `</airhorn:816437322781949972>`                   |
+| Standard Emoji          | Unicode Characters or `:NAME:` | 💯 or `:100:`                                     |
+| Custom Emoji            | `<:NAME:ID>`                   | `<:mmLol:216154654256398347>`                     |
+| Custom Emoji (Animated) | `<a:NAME:ID>`                  | `<a:b1nzy:392938283556143104>`                    |
+| Soundmoji               | `<sound:GUILD_ID:SOUND_ID:>`   | `<sound:1029315212005888060:1147975697353150494>` |
```

Comment:
Add a footnote that guild ID is `0` for default sounds

Reply:
foot? dzi reference???

Reply:
```
^3^ Default sounds use `0` in place for `GUILD_ID`.
```
Good enough?

Reply:
yop